pkg/internal/testing/process/process.go

```diff
@@ -265,6 +266,9 @@ func (ps *State) Stop() error {
 	case <-ps.waitDone:
 		break
 	case <-timedOut:
+		if err := ps.Cmd.Process.Signal(syscall.SIGKILL); err != nil {
```

Comment:
You sure you don't want this? https://github.com/kcp-dev/kcp/pull/2707/commits/1cfbe44a0e9d40c848e15d3ec504d4284262d147

Reply:
Or at least put the kill-negativepid-sigterm up above where we're sending SIGTERM?

Reply:
Hm, wouldn't that kill the entire process group? What if we're only stopping a single piece?

Reply:
Lol all the funky stuff you need to actually operationalize things seems to go back to work by @felixge

Reply:
Um 😄 was writing quickly as I remembered doing this for kcp. I'll defer to you for what the needs are here, in terms of 1 pid vs. multiple.